packages/flutter/lib/src/widgets/sliver_fill.dart

```diff
@@ -0,0 +1,468 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
+
+import '../../widgets.dart';
```

Comment:
"widgets" shouldn't import "widgets". Instead import the file you need from widgets directly.